src/typescript/azure-openapi-validator/rules/DescriptionAvoidsGerunds.ts

```diff
@@ -0,0 +1,36 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+import { MergeStates, OpenApiTypes, rules } from '../rule';
+
+export const DescriptionAvoidsGerunds: string = "DescriptionAvoidsGerunds";
```

Comment:
could you include comments on intent of the rules for each of them? It helps understand what the rule is supposed to do from start, even when the name is not as obvious.

Reply:
Fixed